PowerShellTools.HostService/Resources.resx

```diff
@@ -130,6 +130,9 @@
   <data name="Error_PipelineBusy" xml:space="preserve">
     <value>Pipeline not executed because a pipeline is already executing. Pipelines cannot be executed concurrently.</value>
   </data>
+  <data name="Warning_HigherVersionOnTargetRequiredForDebugging" xml:space="preserve">

```

Comment:
Should include the FW Link to the most recent version [Can use the one when < 3.0 is installed]